### ISSUE TITLE
Error textS "Could not find recipe element" and "Could not fetch site ... " is not translatable in transifex

### ISSUE THREAD

**Issue by cloud2018** [nextcloud/cookbook#1037]
**Description**
Error texts "Could not find recipe element" and "Could not fetch site ... " is not translatable in transifex

**Reproduction**
Steps to reproduce the behavior ("Could not find recipe element"):
1. Go to 'Download recipe from URL' in left sideboard
2. Fill in an URL without an valid recipe
3. See untranslated error text: "Could not find recipe element"

Steps to reproduce the behavior ("Could not find recipe element"):
1. Go to 'Download recipe from URL' in left sideboard
2. Fill in a text wich is not an URL
3. See untranslated error text: "Could not fetch site ... "

**Expected behavior**
Error texts are translated or translatable in transifex

**Versions**
Nextcloud server version: 23
Cookbook version: 0.9.12

![CB_!](https://user-images.githubusercontent.com/34692335/173100860-e89f6e94-bbd2-41c7-a494-02e1a3bf2b62.png)

![CB_!](https://user-images.githubusercontent.com/34692335/173101109-31673fdd-9a76-441b-a422-9a2cda590de9.png)

![CB_!](https://user-images.githubusercontent.com/34692335/173101327-b1a11cda-4c4c-41f3-a0b5-4ded71226589.png)

**Comment by christianlupus** [comment]
The string of the first screenshot I have found in transifex. Which language were you looking at?

The other two strings are probably outdated and cannot be found any longer on transifex. We had a recent change in the source code that reworked the download functionality in a significant way. Thus, the error strings are made more clear and more specific.
I suspect therefore that the last two strings are no longer present in transifex.

Please wait for the next release and report back if the problem still persists.

**Comment by cloud2018** [comment]
yes, "Could not find recipe element" and "Could not fetch site ... " are apparently outdated.

Perhaps the following other 3 strings are untranslated (not tested, only viewed code) :
https://github.com/nextcloud/cookbook/blob/ee393d64587bf5c0f08759f2a6a01193cebdf88d/lib/Service/DbCacheService.php#L126
https://github.com/nextcloud/cookbook/blob/c7131a7315cc691f63ed902f390fecb9f0da0e13/tests/Unit/Controller/RecipeControllerTest.php#L367
https://github.com/nextcloud/cookbook/blob/0bf7832559ba26e645f0d86b66a6d61597e30d16/lib/Db/RecipeDb.php#L44

**Comment by christianlupus** [comment]
The tests (2nd case) must not be translated. The other two are in general valid.
